src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -234,4 +240,12 @@ private void performClaimTaskAction(String taskId, Map<String, String> body) {
             camundaErrorDecoder.decodeException(ex);
         }
     }
+
+    private void performUnclaimTaskAction(String taskId, Map<String, String> body) {
+        try {
+            camundaServiceApi.unclaimTask(authTokenGenerator.generate(), taskId, body);
+        } catch (FeignException ex) {
+            camundaErrorDecoder.decodeException(ex);
```

Comment:
I don't think there's any need to decode it. 
just throw a server error since this endpoint can only return:

204  | Request successful.
400  | Task with given id does not exist.

Since we retrieve the variables before we know the task exists so let's just throw a ServerError exception and assume something has gone wrong.